### PR TITLE
Submit comments on ctrl/cmd-enter

### PR DESCRIPTION
**Pull request by akrolsmir**

### CODE REVIEW

File: web/components/feed/feed-comments.tsx [hunk 1]
```diff
@@ -408,11 +409,9 @@ export function CommentInputTextArea(props: {
     submitComment,
     presetId,
     isSubmitting,
-    submitOnEnter,
+    submitOn,
     replyToUser,
   } = props
-  const isMobile = (useWindowSize().width ?? 0) < 768 // TODO: base off input device (keybord vs touch)
```
Comment (akrolsmir) [inline web/components/feed/feed-comments.tsx:414]: @sipec wondering why we had this at all?

Comment (sipec) [reply web/components/feed/feed-comments.tsx:414]: on touch devices, you should be able to submit on pressing enter, without needing to also press another key
or at least I think that's what it is - this existed before I refactored it way back when

Comment (IanPhilips) [reply web/components/feed/feed-comments.tsx:414]: I think actually it's for the opposite: never allow submitting on enter on mobile, which it seems like noone wants.

Comment (IanPhilips) [reply web/components/feed/feed-comments.tsx:414]: So I would add that back in!

**Pull request merged**